.github/workflows/cont_integration.yml

```
@@ -118,7 +118,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-            toolchain: stable
+            toolchain: 1.78.0
```

```suggestion
            toolchain: 1.63.0
```

Why not 1.63, our MSRV?

https://github.com/bitcoindevkit/bdk/blob/dcd2d4741d46c30a930b6f673cf12016a65edc54/clippy.toml#L1C7-L1C13

cc @danielabrozzoni this might solve our clippy problems.

---

We want the latest lints that are compatible with MSRV, we just don't want them changing unexpectedly.

---

Just saw the discord message now. Makes sense.